src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QueryExecutionService.cs

```diff
@@ -893,13 +894,18 @@ public async Task<ResultSetSubset> InterServiceResultSubset(SubsetParams subsetP
             // Setup the ResultSet updated callback
             ResultSet.ResultSetAsyncEventHandler resultUpdatedCallback = async r =>
             {
+                ExecutionPlanGraph graph = null;
+                if(r.Summary.Complete && r.Summary.SpecialAction.ExpectYukonXMLShowPlan == true)
```

Comment:
nit: add space between 'f' and '('.  Also comparing `ExpectYukonXMLShowPlan` with `true` is somewhat redundant since that is default test in if condition.